src/test/run-make-fulldeps/sanitizer-invalid-cratetype/Makefile

```diff
@@ -3,14 +3,9 @@
 -include ../tools.mk
```

Comment:
I did not expect this test to pass. Is it run for PRs?

Reply:
None of the tests tagged with `# needs-sanitizer-support` are run because the Dockerfile for `x86_64-gnu-llvm-6.0` does not have `--enable-sanitizers` set. The Dockerfile for `x86_64-gnu` does have `--enable-sanitizers`, but is not currently being run on azure.

Reply:
This should pass now - if it were run.

Reply:
cc @rust-lang/infra is there anything that should/could be done here to get these tests running?

Reply:
You can edit `.azure-pipelines/pr.yml` to temporarily run the `x86_64-gnu` builder on this one PR, and then that change would be reverted before approving the PR